docs/platforms/java/common/opentelemetry/usage/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: OpenTelemetry Usage
+sdk: sentry.java.opentelemetry-agent
+description: "Using OpenTelemetry with Sentry."
+sidebar_order: 160
+---
+
+## OpenTelemetry and Sentry
+
+With Sentry’s OpenTelemetry SDK, an OpenTelemetry `Span` becomes a Sentry `Transaction` or `Span`. The first `Span` sent through the Sentry `SpanProcessor` is a `Transaction`, and any child `Span` gets attached to the first `Transaction` upon checking the parent `Span` context. This is true for the OpenTelemetry root `Span` and any top level `Span` in the system. For example, a request sent from frontend to backend will create an OpenTelemetry root `Span` with a corresponding Sentry `Transaction`. The backend request will create a new Sentry `Transaction` for the OpenTelemetry `Span`. The Sentry `Transaction` and `Span` are linked as a trace for navigation and error tracking purposes.
+
+### Manual Instrumentation with OpenTelemetry
+
+If you have the OpenTelemetry SDK in you classpath, you can also instrument your code manually using the OpenTelemetry API as documented [in the OpenTelemetry docs](https://opentelemetry.io/docs/languages/java/api/#span).
+
+A manually created span for HTTP requests needs to declare its `SpanKind` as well as the `HttpAttributes.HTTP_REQUEST_METHOD` attribute, so that `Sentry` can correctly process these:
+
+```java {tabTitle:Java}
+Span span = tracer.spanBuilder("myspan")
+  .setAttribute(HTTP_REQUEST_METHOD, "GET")
+  .setSpanKind(SpanKind.SERVER)
+  .startSpan();
+```
+```kotlin {tabTitle:Kotlin}
+val span = tracer.spanBuilder("myspan")
+  .setAttribute(HTTP_REQUEST_METHOD, "GET")
+  .setSpanKind(SpanKind.SERVER)
+  .startSpan()
+```
+
+### Capturing HTTP Headers
+
+By default OpenTelemetry does not capture any HTTP headers. This, however, can be configured using system properties or environment variables as per OpenTelemetry's configuration documentation [here](https://opentelemetry.io/docs/zero-code/java/agent/instrumentation/http/#capturing-http-request-and-response-headers). Each variable is a comma-separated list of HTTP header names that should be captured.
```

Comment:
Looks like something I forgot initially when creating this section. But should we add a disclaimer here to look out for PII in the headers?